core/src/main/java/org/commonjava/maven/ext/core/groovy/BaseScriptUtils.java

```diff
@@ -73,6 +73,15 @@ public void inlineProperty ( Project currentProject, ProjectRef groupArtifact )
                               a.getValue().setVersion(
                                               PropertyResolver.resolvePropertiesUnchecked( getSession(), currentProject.getInheritedList(), a.getValue().getVersion() ) );
                           } );
+            currentProject.getResolvedPlugins( getSession() )
```

Comment:
I wonder if we should cover managed plugins as well as plugins (given that the function currently covers dependencies and managed dependencies) ?

Reply:
Good point, I will try to add managed plugins as well.

Reply:
@rnc I implemented this change and pushed a new commit to the PR.